packages/did/src/DidDetails/FullDidDetails.ts

```diff
@@ -34,6 +34,7 @@ const methodMapping: Record<string, VerificationKeyRelationship | undefined> = {
   'did.submitDidCall': undefined,
   didLookup: 'authentication',
   didName: 'authentication',
+  score:'authentication',
```

Comment:
Should be assertionMethod here. Authentication should only be used for DID management-related operations.